packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue

```diff
@@ -0,0 +1,85 @@
+<script lang="ts" setup>
+import { createListbox } from "@sit-onyx/headless";
+import type { OnyxListboxOptionProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxListboxOptionProps>(), {
+  disabled: false,
+});
+
+const emit = defineEmits<{
+  "update:modelValue": [value: boolean];
+}>();
+
+const {
+  elements: { option },
+} = createListbox({
+  label: "",
+  onSelect: () => emit("update:modelValue", !props.modelValue),
+});
+</script>
+
+<template>
+  <!-- eslint-disable-next-line vuejs-accessibility/no-static-element-interactions --
+  the required attributes are set by the v-bind. The eslint rule does not check the v-bind so we need
+  to disable it here.
+  -->
+  <li
+    v-bind="
+      option({
+        id: props.label,
+        label: props.label,
+        selected: props.modelValue,
+        disabled: props.disabled,
+      })
+    "
+    class="onyx-listbox-option"
+    :class="{ 'onyx-listbox-option--selected': props.modelValue }"
+    :disabled="props.disabled"
```

Comment:
AFAIK `disabled` doesn't have an effect here? 

```suggestion
```

Reply:
Nope it is a leftover from my previous button approach. I removed it